ts/src/main/scala/ch/epfl/ts/engine/MarketFXSimulator.scala

```diff
@@ -15,7 +15,7 @@ class MarketFXSimulator(marketId: Long, rules: ForexMarketRules) extends MarketS
     // TODO
 
     case limitAsk: LimitAskOrder =>
-    // TODO
+    // TODO - add limit order resolvation based on given quotes
```

Comment:
And no, because they don't care about `Quote`s at all and only keep order books. If we use their strategy, we'll get two divergent symbol prices, one for `LimitOrder`s and one for `MarketOrder`s.